docs/develop/advanced/17-autocli.md

```diff
@@ -0,0 +1,196 @@
+---
+sidebar_position: 1
+---
+
+# AutoCLI
```

Comment:
ditto on the .gitignore?

Reply:
This should never have been committed there :/
This is copied from client/v2/README.md